pkg/ingestr/operator.go

```diff
@@ -58,35 +58,38 @@ func (o BasicOperator) Run(ctx context.Context, ti scheduler.TaskInstance) error
 	if !ok {
 		return errors.New("source connection not configured")
 	}
+
 	sourceConnection, err := o.conn.GetConnection(sourceConnectionName)
 	if err != nil {
 		return fmt.Errorf("source connection %s not found", sourceConnectionName)
 	}
+
 	sourceURI, err := sourceConnection.(pipelineConnection).GetIngestrURI()
 	if err != nil {
 		return errors.New("could not get the source uri")
 	}
+
 	sourceTable, ok := ti.GetAsset().Parameters["source_table"]
 	if !ok {
 		return errors.New("source table not configured")
 	}
 
-	destConnectionName, ok := ti.GetAsset().Parameters["destination_connection"]
-	if !ok {
-		return errors.New("destination connection not configured")
+	destConnectionName := ti.GetAsset().Connection
```

Comment:
this wouldn't work, we need this to be a function that can get the pipeline-level defaults if needed as well